internal/sdkv2provider/resource_cloudflare_pages_project_test.go

```diff
@@ -230,7 +230,6 @@ func TestAccCloudflarePagesProject_DeploymentConfig(t *testing.T) {
 
 	rnd := generateRandomResourceName()
 	name := "cloudflare_pages_project." + rnd
-	accountID := os.Getenv("CLOUDFLARE_ACCOUNT_ID")
```

Comment:
these need to stay to use the correct value

Reply:
Added back

`accountID` is available globally from the access group test
https://github.com/cloudflare/terraform-provider-cloudflare/blob/master/internal/sdkv2provider/resource_cloudflare_access_group_test.go#L72

Reply:
yep - that will disappear in the near future. it shouldn't be globally available.